Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -51,7 +51,7 @@
     <Compile Include="**\*.netstandard.*.cs" />
   </ItemGroup>
   <ItemGroup Condition=" $(TargetFramework.StartsWith('uap10.0')) ">
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.0.6" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.1.5" />
```

Comment:
do we want to bump to this? I assume it is okay? will it align with templates and does it require specific versions of .NET Core to be installed?

Reply:
This is not related to .net core (MS just used that name for some reason). This is the new version in the templates: https://github.com/xamarin/xamarin-templates/search?q=UniversalWindowsPlatform&unscoped_q=UniversalWindowsPlatform

Reply:
Should just sync with @pierceboggan to make sure our templates use this version too.